UX/ui-service/MyOrders/design.md

```diff
@@ -0,0 +1,42 @@
+# My Orders
+
+- My Orders is a primary navigation item
+- Administrators
+  - View and approve or deny orders for other users
+- End Users
+  - View status of placed orders
+
+## Admin user
+
+### Review Orders
+![Image Title/Mockup Name](img/Order_01.png)
+- The Admin User sees all orders that have been placed.
+
+### Approve Order
+![Image Title/Mockup Name](img/Order_02.png)
+- The Administrator can check and approve an order or its component pieces
+
+
+#### Implementation Details
+  - The orders list should use the [List View with simple expansion](https://www.patternfly.org/pattern-library/content-views/list-view/#/api#expanding-rows) pattern
+
+## End user
+
+### View Orders
+![Image Title/Mockup Name](img/Order_03.png)
+- The End User sees only orders that they have placed
+
+### View Requests
+![Image Title/Mockup Name](img/Order_04.png)
+- They can check the status of their currently pending orders and view all past orders and their component requests
+
+### View Request
```

Comment:
@cshinn Can you check in with @beanh66 on this ? 
Pending requests should be able to be edited by the end user.  It would be nice to show this flow as well, documenting where the "Edit" button is, and then what screen shows up as well as the location of "Save" & "Cancel"